app/Imports/XlsformTemplateLanguageImport.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Imports;
+
+use App\Models\LanguageString;
+use App\Models\SurveyRow;
+use App\Models\LanguageStringType;
+use Maatwebsite\Excel\Concerns\ToModel;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+
+class XlsformTemplateLanguageImport implements ToModel, WithHeadingRow
+{
+    protected $templateId;
+    protected $templateLanguageId;
+
+    public function __construct($templateId, $templateLanguageId)
```

Comment:
Probably only need to pass in the XlsformTemplateLanguage entry here, as the template can be got from the relationship?